pkg/generated/openapi/doc.go

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
@smarterclayton anyway I think we should separate federation types in another PR. I think it is not related to this PR.